src/application/machines.ts

```diff
@@ -87,15 +87,14 @@ export function createDevtoolsMachine({ actions }: { actions: Actions }) {
             timeout: "timedout",
             clientNotFound: "notFound",
           },
-          entry: ["unsubscribeFromAll"],
+          entry: ["cancelRequestInterval"],
         },
         timedout: {},
         notFound: {
           on: {
             retry: "retrying",
             connect: "connected",
           },
-          entry: "unsubscribeFromAll",
```

Comment:
With this now called `cancelRequestInterval`, this action made no sense here since the only time we'll have a request interval is when we start one, which happens when we enter the `connected` state. There is no path from `connected` to `notFound`, so this will never have any effect.